### PR TITLE
[15950] Tutorial - External Locators

### PR DESCRIPTION
**Pull request by Mario-DL**
This PR adds a tutorial for leveraging Fast DDS' external locators feature in Vulcanexus.

Cannot be merged before:

*  eprosima/Fast-DDS#3079
* #86

### CODE REVIEW

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (Mario-DL) [reply docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:187]: I moved it before the XML file description, after the line referring to the example above

`Following the example above ....
.. note:: 
....`

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:98]:
```suggestion
                                            <!-- Container 1 peer-->
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:108]:
```suggestion
                                            <!--
                                                Local network DDS default multicast to discover
                                                other participants in the same LAN,
                                                using External Locators, or not
                                            -->
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:72]: I'd move all the XML files to several resource files (like in #86) and include them with literal include.

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:164]: Same as above

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:182]:
```suggestion
This section provides with step-by-step instructions for setting up the example scenario described in previous sections.
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:235]:
```suggestion
Finally, export the environment variable pointing to the Profiles.xml file, source Vulcanexus environment and run the ros2 example nodes.
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:248]:
```suggestion
                    export FASTRTPS_DEFAULT_PROFILES_FILE=/Profiles.xml # Or the Profiles.xml file location
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:268]:
```suggestion
                        export FASTRTPS_DEFAULT_PROFILES_FILE=/Profiles.xml # Or the Profiles.xml file location
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:273]:
```suggestion
A message `Hello World: [count]` should start printing in the talker's container terminal while both listeners keep receiving it, in their respective container consoles, as follows:
```

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:278]: Nitpick, the host 2 listener seems to be receiving data before the host 1 talker sends it

**Review by EduPonz — CHANGES_REQUESTED** [review]
I'd like to congratulate you on the video!

**Comment by EduPonz** [comment]
I've rebased this on top of #86 again since there I had to add the video as raw HTML instead of through the extension because that was breaking the latex compilation on RTD. I've enabled a [build on this branch](https://docs.vulcanexus.org/en/docs-ext_loc_tutorial/rst/tutorials/core/deployment/external_locators/external_locators.html) so we can see how it looks online before merging.

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (EduPonz) [inline docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:277]:
```suggestion
    <video width=100% height=auto autoplay loop controls muted>
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/rst/tutorials/core/deployment/external_locators/external_locators.rst
Comment (Mario-DL) [reply docs/rst/tutorials/core/deployment/external_locators/external_locators.rst:22]: Thanks !

**Review by EduPonz — APPROVED** [review]
Tentatively approving as the docs look very nice to me. However, I'll not merge it until I can run the tutorial once the following is merged:

*  eprosima/Fast-DDS#3079

**Review by EduPonz — APPROVED** [review]

**Pull request merged**
